CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing Guide
+
+Thank you for your efforts to improve the fapolicy-analyzer project. This guide
+will help you regarding various aspects like submitting issues, contributing a
+feature, etc.
+
+## Developers
+
+### Python bindings
+
+We write python bindings using [PyO3](https://github.com/PyO3/pyo3) and [setuptools_rust](https://setuptools-rust.readthedocs.io/en/latest/).
+
+We use [pipenv](https://pipenv.pypa.io/en/latest/) for creating a sand-boxed development environment.  To install `pipenv` into your home directory:
+
+```{shell}
+pip3 install --user pipenv
+```
+
+To build and install the bindings run the following from the top level directory of this repository:
+
+```{shell}
+make shell
+```
+
+which under the hood is executing the following with the `develop` argument to setup.py:
```

Comment:
I think cut out this bit about what happens under the hood

Reply:
Done